### DISCUSSION TITLE
unable to save a complete profile

### DISCUSSION THREAD

**Discussion by yip623** [Eugeny/tabby#6141] [General]
version 1.0.174
I have tried to entry a huge amount of connection into the profile. I found the case that if entering 1 or 2 connections, I have to exit the application, I can see the saved new connections when I reopen the application. However, when i enter a lot of connections, it is only saved the recent few entries after reopen the application.

**Comment by Eugeny** [comment]
Do you mean Settings -> Profiles and Connections or the Recent Connections (shown below)?
<img width="496" alt="image" src="https://user-images.githubusercontent.com/161476/162898597-0ca5b0b7-1798-401d-89f6-4166315493d3.png">
The latter only saves the 3 last connections.

**Reply by yip623** [reply]
Thanks for your quick response.
The problem is not related to "Recent Connections"

I have created multiple new "SSH connection" one by one, let say 20 profiles. Then I close the application and open it again. I can only find 2 or 3 connections in the profile, and the other 17 or 18 disappear.
![image](https://user-images.githubusercontent.com/25526574/162904860-49fb2cb9-e03f-4abe-8369-faa5aae8d78c.png)

Currently, i have to create one SSH connection and then reopen the application. But I still have a hundard of connection required to be created.

**Reply by yip623** [reply]
The same case happens in both Windows 10 and Ubuntu 20

**Reply by zzXsrgdo** [reply]
I'm having the same problem in Windows 11.

**Comment by yip623** [comment]
the problem still exists in v1.0.175

**Reply by Redika-ak** [reply]
the problem still exist in V1.0.232 on may 2026

**Comment by Tode** [comment]
I have the same issue. Saving does nothing
